File: Python/1 - Iniciante/1072.py
# coding UTF-8
# Funcional no URI

N = int(input())
dentro = int(0)
fora = int(0)
for i in range(1, N+1):
    X = int(input())
    if 10 <= X <= 20:
        dentro += 1
    else:
        fora += 1
print('{} in'.format(dentro))
print('{} out'.format(fora))

File: go/1072.go
// Funcional no beecrowd
package main

import (
	"fmt"
)

func main() {
	var N int

	fmt.Scan(&N)

	dentro := 0
	fora := 0

	for i := 1; i <= N; i++ {
		var X int
		fmt.Scan(&X)
		if 10 <= X && X <= 20 {
			dentro++
		} else {
			fora++
		}
	}

	fmt.Printf("%d in\n", dentro)
	fmt.Printf("%d out\n", fora)
}
